test/transaction_rejection.js

```diff
@@ -25,7 +25,7 @@ describe("Transaction rejection", function() {
   var estimateGasContractAbi;
   var EstimateGasContract;
   var estimateGasContractAddress;
-  var source = fs.readFileSync(path.join(__dirname, "EstimateGas.sol"), "utf8");
+  var source = fs.readFileSync(path.join(__dirname, "/contracts/gas/EstimateGas.sol"), "utf8");
```

Comment:
same comment as above re: `path.join` & path separators

Reply:
Can change for now but will be removed in a future PR